### DISCUSSION TITLE
Organization members should be able to see Org self-hosted runners' status

### DISCUSSION THREAD

**Discussion by daltonv** [community/community#13691] [Actions]
Currently org members cannot see the organization self-hosted runners at all unless they have Admin access to a repository. This makes it hard for most users to actually determine if a self-hosted runner is being used or is completely down. 

Devs shouldn't have to contact a repo Admin or Org Owner to figure out why there workflow is taking so long to find a runner.

**Comment by ethomson** [comment]
Thanks @daltonv - I appreciate the feedback. And I agree. We'll look into some ways that we can improve this experience.

I'll make sure that this is on the backlog.
